src/behavior.js

```diff
@@ -32,13 +32,7 @@ const Behavior = MarionetteObject.extend({
     // to the view.
     this.view = view;
 
-    if (this.defaults) {
-      deprecate('Behavior defaults are deprecated. For similar functionality set options on the Behavior class.');
```

Comment:
this might be the only use of `deprecate` so the import could be removed.

Reply:
Done